### PR TITLE
3537 - Hide 'any' option from filters

### LINKED ISSUE #3537: Remove the "any" option from filter (3 pts)
The any filter aggregations have some bugs, and fixing them may require more time than what is worth, so for now it's been decided to remove it.

 - ~Remove the any option from filters and any code in client side related to it.~
 - ~Remove any backend code to calculate aggregations.~
 - ~Remove the filter from the queryBuilder ?~
 -  Hide it

### PR DESCRIPTION
**Pull request by Zasa-san**
fixes #3537 

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

File: app/react/Forms/components/MultiSelect.tsx [hunk 1]
```diff
@@ -317,24 +317,27 @@ abstract class MultiSelectBase<ValueType> extends Component<
...
-            this.checked(option) === SelectStates.PARTIAL ? ' partial' : ''
-          }`}
-          value={option[optionsValue]}
-          id={prefix + option[optionsValue]}
-          onChange={this.change.bind(this, option)}
-          checked={this.checked(option) !== SelectStates.OFF}
-        />
-        {this.label(option)}
-      </li>
+      isNotAnyOption && (
```
Comment (mfacar) [inline app/react/Forms/components/MultiSelect.tsx:322]: @Zasa-san why did you decide to exclude the 'any' option here? Given the issue is related only to the filters, isn't the  [FiltersFromProperties](https://github.com/huridocs/uwazi/blob/109cd29aae9732f05ade623a7dd0e068784754cd/app/react/Library/components/FiltersFromProperties.js#L35) component a better place to exclude the option?

**Review by mfacar — CHANGES_REQUESTED** [review]
@Zasa-san I tested the functionality and I couldn't find any impact. However, I think that the exclusion could be implemented at the use level of the component.

File: app/react/Forms/components/MultiSelect.tsx (hunk 1 above)
Comment (Zasa-san) [reply app/react/Forms/components/MultiSelect.tsx:322]: It makes sense to exclude it in the FiltersFromProperties component.

**Review by mfacar — APPROVED** [review]
LGTM! Nice cleanup of the test file 👍

**Pull request merged**